### ISSUE TITLE
returning 0/1 student result in an error.

### ISSUE THREAD

**Issue by rnieuweveen** [Tools4everBV/HelloID-Conn-Prov-Source-Topicus-ParnasSys#6]
When parnassys return 1 student it will count the properties of the single object. also with 0 students the check isn't working.

in my case fixed with replacing all:
`$****.count` for `($****| measure-object).count`

also when parnassys returning 1 student it will create an hashtable instead of creating an array and the script will try adding the hashtable to an array of objects.

in my case fixed with replacing:
`$leerLingen , $leerLingenReturnNode = $null ` for ` $leerLingen , $leerLingenReturnNode = @()`

and:
`$leerLingenPreviousYear , $leerLingenPreviousYearReturnNode = @()`

**Comment by Rick-Jongbloed** [comment]
HI Rick, thanks for reporting this :-) Could you please make a PR with your changes?

**Comment by Rick-Jongbloed** [comment]
@JeroenBL  waarom heb je dit weer opnieuw geopend? Is het probleem toch niet gefixt?

**Comment by JeroenBL** [comment]
Nope... Patrick kwam laatst op de lijn dat het probleem nog steeds speelde. Dus we moeten er nog even een keer induiken.
